README.md

```diff
@@ -1,2 +1,35 @@
-# angular-injecting-a-service-lab
-<p data-visibility='hidden'>View <a href='https://learn.co/lessons/angular-injecting-a-service-lab' title='angular-injecting-a-service-lab'>angular-injecting-a-service-lab</a> on Learn.co and start learning to code for free.</p>
+# Injecting a Service
+
+## Objectives
+
+- Create a $timeout that updates the View
+
+## Instructions
+
+Let's inject one of Angular's built in services into our controller. 
+
+For this lab, we're going to make use of `$timeout`. `$timeout` allows us to create timeouts (similar to `setTimeout`) in our applications whilst ensuring that any changes to our model done inside a timeout gets reflected in the view.
```

Comment:
include a basic example of how to use `$timeout` here  so students can see the syntax before applying it on their own.